packages/block-library/src/paragraph/deprecated.js

```diff
@@ -57,12 +51,103 @@ const blockAttributes = {
 		type: 'string',
 		enum: [ 'ltr', 'rtl' ],
 	},
+	style: {
+		type: 'object',
+	},
+};
+
+const migrateCustomColors = ( attributes ) => {
```

Comment:
Would it make sense to share this migration function across our blocks?

Reply:
yes, I duplicated for now but yes it's always the same.